{"nodes":[{"pos":[603,786],"content":"Represents the exception that is thrown when you try to perform an operation on a <bpt id=\"p1\">&lt;xref href=\"System.Data.DataRow\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> that is not in a <bpt id=\"p2\">&lt;xref href=\"System.Data.DataTable\"&gt;</bpt><ept id=\"p2\">&lt;/xref&gt;</ept>.","needQuote":true,"needEscape":true,"source":"Represents the exception that is thrown when you try to perform an operation on a <xref href=\"System.Data.DataRow\"></xref> that is not in a <xref href=\"System.Data.DataTable\"></xref>."},{"pos":[799,1519],"content":"The RowNotInTableException is thrown when invoking the following methods on a row that has been deleted with either the <xref:System.Data.DataRow.Delete%2A> or the <xref:System.Data.DataRowCollection.Remove%2A?displayProperty=fullName> method.      -   <xref:System.Data.DataRow.AcceptChanges%2A?displayProperty=fullName>      -   <xref:System.Data.DataRow.GetChildRows%2A?displayProperty=fullName>      -   <xref:System.Data.DataRow.GetParentRow%2A?displayProperty=fullName>      -   <xref:System.Data.DataRow.GetParentRows%2A?displayProperty=fullName>      -   <xref:System.Data.DataRow.RejectChanges%2A?displayProperty=fullName>      -   <xref:System.Data.DataRow.SetParentRow%2A?displayProperty=fullName>","needQuote":false,"needEscape":true,"extradata":"MT","nodes":[{"content":"The RowNotInTableException is thrown when invoking the following methods on a row that has been deleted with either the <xref:System.Data.DataRow.Delete%2A> or the <xref:System.Data.DataRowCollection.Remove%2A?displayProperty=fullName> method.      -   <xref:System.Data.DataRow.AcceptChanges%2A?displayProperty=fullName>      -   <xref:System.Data.DataRow.GetChildRows%2A?displayProperty=fullName>      -   <xref:System.Data.DataRow.GetParentRow%2A?displayProperty=fullName>      -   <xref:System.Data.DataRow.GetParentRows%2A?displayProperty=fullName>      -   <xref:System.Data.DataRow.RejectChanges%2A?displayProperty=fullName>      -   <xref:System.Data.DataRow.SetParentRow%2A?displayProperty=fullName>","pos":[0,708],"nodes":[{"content":"The RowNotInTableException is thrown when invoking the following methods on a row that has been deleted with either the &lt;xref:System.Data.DataRow.Delete%2A&gt; or the &lt;xref:System.Data.DataRowCollection.Remove%2A?displayProperty=fullName&gt; method.","pos":[0,243],"source":"The RowNotInTableException is thrown when invoking the following methods on a row that has been deleted with either the <xref:System.Data.DataRow.Delete%2A> or the <xref:System.Data.DataRowCollection.Remove%2A?displayProperty=fullName> method."},{"content":"-   &lt;xref:System.Data.DataRow.AcceptChanges%2A?displayProperty=fullName&gt;      -   &lt;xref:System.Data.DataRow.GetChildRows%2A?displayProperty=fullName&gt;      -   &lt;xref:System.Data.DataRow.GetParentRow%2A?displayProperty=fullName&gt;      -   &lt;xref:System.Data.DataRow.GetParentRows%2A?displayProperty=fullName&gt;      -   &lt;xref:System.Data.DataRow.RejectChanges%2A?displayProperty=fullName&gt;      -   &lt;xref:System.Data.DataRow.SetParentRow%2A?displayProperty=fullName&gt;","pos":[249,708],"source":"      -   <xref:System.Data.DataRow.AcceptChanges%2A?displayProperty=fullName>      -   <xref:System.Data.DataRow.GetChildRows%2A?displayProperty=fullName>      -   <xref:System.Data.DataRow.GetParentRow%2A?displayProperty=fullName>      -   <xref:System.Data.DataRow.GetParentRows%2A?displayProperty=fullName>      -   <xref:System.Data.DataRow.RejectChanges%2A?displayProperty=fullName>      -   <xref:System.Data.DataRow.SetParentRow%2A?displayProperty=fullName>"}]}]},{"pos":[2693,2789],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.Data.RowNotInTableException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.Data.RowNotInTableException\"></xref> class."},{"pos":[3390,3512],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.Data.RowNotInTableException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class with the specified string.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.Data.RowNotInTableException\"></xref> class with the specified string."},{"pos":[3651,3702],"content":"The string to display when the exception is thrown.","needQuote":true,"needEscape":true,"nodes":[{"content":"The string to display when the exception is thrown.","pos":[0,51]}]},{"pos":[4469,4596],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.Data.RowNotInTableException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class with serialization information.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.Data.RowNotInTableException\"></xref> class with serialization information."},{"pos":[4872,4936],"content":"The data that is required to serialize or deserialize an object.","needQuote":true,"needEscape":true,"nodes":[{"content":"The data that is required to serialize or deserialize an object.","pos":[0,64]}]},{"pos":[5032,5109],"content":"Description of the source and destination of the specified serialized stream.","needQuote":true,"needEscape":true,"nodes":[{"content":"Description of the source and destination of the specified serialized stream.","pos":[0,77]}]},{"pos":[5698,5900],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.Data.RowNotInTableException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class with a specified error message and a reference to the inner exception that is the cause of this exception.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.Data.RowNotInTableException\"></xref> class with a specified error message and a reference to the inner exception that is the cause of this exception."},{"pos":[6077,6138],"content":"The error message that explains the reason for the exception.","needQuote":true,"needEscape":true,"nodes":[{"content":"The error message that explains the reason for the exception.","pos":[0,61]}]},{"pos":[6212,6410],"content":"The exception that is the cause of the current exception, or a null reference (<bpt id=\"p1\">&lt;xref uid=\"langword_csharp_Nothing\" name=\"Nothing\" href=\"\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> in Visual Basic) if no inner exception is specified.","needQuote":true,"needEscape":true,"source":"The exception that is the cause of the current exception, or a null reference (<xref uid=\"langword_csharp_Nothing\" name=\"Nothing\" href=\"\"></xref> in Visual Basic) if no inner exception is specified."}],"content":"### YamlMime:ManagedReference\nitems:\n- uid: System.Data.RowNotInTableException\n  id: RowNotInTableException\n  children:\n  - System.Data.RowNotInTableException.#ctor\n  - System.Data.RowNotInTableException.#ctor(System.String)\n  - System.Data.RowNotInTableException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  - System.Data.RowNotInTableException.#ctor(System.String,System.Exception)\n  langs:\n  - csharp\n  name: RowNotInTableException\n  nameWithType: RowNotInTableException\n  fullName: System.Data.RowNotInTableException\n  type: Class\n  summary: Represents the exception that is thrown when you try to perform an operation on a <xref href=\"System.Data.DataRow\"></xref> that is not in a <xref href=\"System.Data.DataTable\"></xref>.\n  remarks: \"The RowNotInTableException is thrown when invoking the following methods on a row that has been deleted with either the <xref:System.Data.DataRow.Delete%2A> or the <xref:System.Data.DataRowCollection.Remove%2A?displayProperty=fullName> method.  \\n  \\n-   <xref:System.Data.DataRow.AcceptChanges%2A?displayProperty=fullName>  \\n  \\n-   <xref:System.Data.DataRow.GetChildRows%2A?displayProperty=fullName>  \\n  \\n-   <xref:System.Data.DataRow.GetParentRow%2A?displayProperty=fullName>  \\n  \\n-   <xref:System.Data.DataRow.GetParentRows%2A?displayProperty=fullName>  \\n  \\n-   <xref:System.Data.DataRow.RejectChanges%2A?displayProperty=fullName>  \\n  \\n-   <xref:System.Data.DataRow.SetParentRow%2A?displayProperty=fullName>\"\n  example:\n  - \"The following example creates a <xref:System.Data.DataTable> with one <xref:System.Data.DataColumn> and 10 <xref:System.Data.DataRow> objects. After deleting a row, the <xref:System.Data.DataRow.AcceptChanges%2A> method is called causing the RowNotInTableException to be thrown.  \\n  \\n [!code-vb[Classic WebData RowNotInTableException Example#1](~/add/codesnippet/visualbasic/t-system.data.rownotinta_1.vb)]\\n [!code-cs[Classic WebData RowNotInTableException Example#1](~/add/codesnippet/csharp/t-system.data.rownotinta_1.cs)]\"\n  syntax:\n    content: 'public class RowNotInTableException : System.Data.DataException'\n  inheritance:\n  - System.Exception\n  - System.SystemException\n  - System.Data.DataException\n  implements: []\n  inheritedMembers: []\n  platform:\n  - net462\n- uid: System.Data.RowNotInTableException.#ctor\n  id: '#ctor'\n  parent: System.Data.RowNotInTableException\n  langs:\n  - csharp\n  name: RowNotInTableException()\n  nameWithType: RowNotInTableException.RowNotInTableException()\n  fullName: System.Data.RowNotInTableException.RowNotInTableException()\n  type: Constructor\n  assemblies:\n  - System.Data\n  namespace: System.Data\n  summary: Initializes a new instance of the <xref href=\"System.Data.RowNotInTableException\"></xref> class.\n  syntax:\n    content: public RowNotInTableException ();\n    parameters: []\n  overload: System.Data.RowNotInTableException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.Data.RowNotInTableException.#ctor(System.String)\n  id: '#ctor(System.String)'\n  parent: System.Data.RowNotInTableException\n  langs:\n  - csharp\n  name: RowNotInTableException(String)\n  nameWithType: RowNotInTableException.RowNotInTableException(String)\n  fullName: System.Data.RowNotInTableException.RowNotInTableException(String)\n  type: Constructor\n  assemblies:\n  - System.Data\n  namespace: System.Data\n  summary: Initializes a new instance of the <xref href=\"System.Data.RowNotInTableException\"></xref> class with the specified string.\n  syntax:\n    content: public RowNotInTableException (string s);\n    parameters:\n    - id: s\n      type: System.String\n      description: The string to display when the exception is thrown.\n  overload: System.Data.RowNotInTableException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.Data.RowNotInTableException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'\n  parent: System.Data.RowNotInTableException\n  langs:\n  - csharp\n  name: RowNotInTableException(SerializationInfo,StreamingContext)\n  nameWithType: RowNotInTableException.RowNotInTableException(SerializationInfo,StreamingContext)\n  fullName: System.Data.RowNotInTableException.RowNotInTableException(SerializationInfo,StreamingContext)\n  type: Constructor\n  assemblies:\n  - System.Data\n  namespace: System.Data\n  summary: Initializes a new instance of the <xref href=\"System.Data.RowNotInTableException\"></xref> class with serialization information.\n  syntax:\n    content: protected RowNotInTableException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);\n    parameters:\n    - id: info\n      type: System.Runtime.Serialization.SerializationInfo\n      description: The data that is required to serialize or deserialize an object.\n    - id: context\n      type: System.Runtime.Serialization.StreamingContext\n      description: Description of the source and destination of the specified serialized stream.\n  overload: System.Data.RowNotInTableException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.Data.RowNotInTableException.#ctor(System.String,System.Exception)\n  id: '#ctor(System.String,System.Exception)'\n  parent: System.Data.RowNotInTableException\n  langs:\n  - csharp\n  name: RowNotInTableException(String,Exception)\n  nameWithType: RowNotInTableException.RowNotInTableException(String,Exception)\n  fullName: System.Data.RowNotInTableException.RowNotInTableException(String,Exception)\n  type: Constructor\n  assemblies:\n  - System.Data\n  namespace: System.Data\n  summary: Initializes a new instance of the <xref href=\"System.Data.RowNotInTableException\"></xref> class with a specified error message and a reference to the inner exception that is the cause of this exception.\n  syntax:\n    content: public RowNotInTableException (string message, Exception innerException);\n    parameters:\n    - id: message\n      type: System.String\n      description: The error message that explains the reason for the exception.\n    - id: innerException\n      type: System.Exception\n      description: The exception that is the cause of the current exception, or a null reference (<xref uid=\"langword_csharp_Nothing\" name=\"Nothing\" href=\"\"></xref> in Visual Basic) if no inner exception is specified.\n  overload: System.Data.RowNotInTableException.#ctor*\n  exceptions: []\n  platform:\n  - net462\nreferences:\n- uid: System.Data.DataException\n  isExternal: false\n  name: System.Data.DataException\n- uid: System.Data.RowNotInTableException.#ctor\n  parent: System.Data.RowNotInTableException\n  isExternal: false\n  name: RowNotInTableException()\n  nameWithType: RowNotInTableException.RowNotInTableException()\n  fullName: System.Data.RowNotInTableException.RowNotInTableException()\n- uid: System.Data.RowNotInTableException.#ctor(System.String)\n  parent: System.Data.RowNotInTableException\n  isExternal: false\n  name: RowNotInTableException(String)\n  nameWithType: RowNotInTableException.RowNotInTableException(String)\n  fullName: System.Data.RowNotInTableException.RowNotInTableException(String)\n- uid: System.String\n  parent: System\n  isExternal: true\n  name: String\n  nameWithType: String\n  fullName: System.String\n- uid: System.Data.RowNotInTableException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  parent: System.Data.RowNotInTableException\n  isExternal: false\n  name: RowNotInTableException(SerializationInfo,StreamingContext)\n  nameWithType: RowNotInTableException.RowNotInTableException(SerializationInfo,StreamingContext)\n  fullName: System.Data.RowNotInTableException.RowNotInTableException(SerializationInfo,StreamingContext)\n- uid: System.Runtime.Serialization.SerializationInfo\n  parent: System.Runtime.Serialization\n  isExternal: false\n  name: SerializationInfo\n  nameWithType: SerializationInfo\n  fullName: System.Runtime.Serialization.SerializationInfo\n- uid: System.Runtime.Serialization.StreamingContext\n  parent: System.Runtime.Serialization\n  isExternal: true\n  name: StreamingContext\n  nameWithType: StreamingContext\n  fullName: System.Runtime.Serialization.StreamingContext\n- uid: System.Data.RowNotInTableException.#ctor(System.String,System.Exception)\n  parent: System.Data.RowNotInTableException\n  isExternal: false\n  name: RowNotInTableException(String,Exception)\n  nameWithType: RowNotInTableException.RowNotInTableException(String,Exception)\n  fullName: System.Data.RowNotInTableException.RowNotInTableException(String,Exception)\n- uid: System.Exception\n  parent: System\n  isExternal: true\n  name: Exception\n  nameWithType: Exception\n  fullName: System.Exception\n- uid: System.Data.RowNotInTableException.#ctor*\n  parent: System.Data.RowNotInTableException\n  isExternal: false\n  name: RowNotInTableException\n  nameWithType: RowNotInTableException.RowNotInTableException\n"}